notify/src/windows.rs

```diff
@@ -295,19 +295,17 @@ fn start_read(rd: &ReadData, event_handler: Arc<Mutex<dyn EventHandler>>, handle
             monitor_subdir,
             flags,
             &mut 0u32 as *mut u32, // not used for async reqs
-            &mut *overlapped as *mut OVERLAPPED,
+            (&mut **overlapped) as *mut OVERLAPPED,
             Some(handle_event),
         );
 
         if ret == 0 {
             // error reading. retransmute request memory to allow drop.
-            // allow overlapped to drop by omitting forget()
+            // allow overlapped to drop by omitting drop()
```

Comment:
```suggestion
            // Because of the error, ownership of `overlapped` was not passed
            // over to `ReadDirectoryChangesW`.
            // So we can claim ownership back
            let _overlapped = ManuallyDrop::into_inner(overlapped);
```